<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FILE 선택기</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Folder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Not yet selected</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>finish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>select_sourcefolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>setSourceFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>selectitem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>finish()</slot>
  <slot>select_sourcefolder()</slot>
  <slot>selectitem()</slot>
  <slot>setSourceFile()</slot>
 </slots>
</ui>
